### PR TITLE
chore: query org with message extension sample suit for v3

### PR DESCRIPTION
**Pull request by wenytang-ms**

### CODE REVIEW

File: query-org-user-with-message-extension-sso/.fx/states/local.userdata [hunk 1]
```diff
@@ -0,0 +1 @@
+fx-resource-bot.botPassword=crypto_3404b79b03d37a3df0f18fdb5f3accbacee1e4012d49b310297a49de3138a8a564af9dc93c2658315ddfdbcd1ba90656ee2e4ab2da9487278c20f686c9c71ff723585f70fc07a923cc70dd1dda73dbabfd001ef26063351db527167dee0fde39c326b7f05d710c7bd8f9e568a1dc0896316e62e640a0beb532dfb81ae9b300b40197e7fa659c741f
```
Comment (SLdragon) [inline query-org-user-with-message-extension-sso/.fx/states/local.userdata:1]: Should not commit this file and .fx folder?

Comment (wenytang-ms) [reply query-org-user-with-message-extension-sso/.fx/states/local.userdata]: thanks. removed

File: query-org-user-with-message-extension-sso/web.config [hunk 2]
```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (SLdragon) [inline query-org-user-with-message-extension-sso/web.config:1]: Is this file really needed?

Comment (wenytang-ms) [reply query-org-user-with-message-extension-sso/web.config:1]: this file is required in IIS node project.
https://github.com/OfficeDev/TeamsFx/wiki/How-to-configure-WEBSITE_RUN_FROM_PACKAGE-in-Azure-App-Service

**Review by SLdragon — APPROVED** [review]
